pallets/account-linker/src/tests.rs

```diff
@@ -53,13 +54,13 @@ fn test_btc_link() {
 		// Generate pay-to-pubkey-hash address
 		let address = Address::p2pkh(&public_key, Network::Bitcoin);
 
-		let account: u64 = 5;
-		let block_number: u64 = 99999;
+		let account: AccountId32 = Default::default();
```

Comment:
default value is a too special one, better to have a random or initialized one

Reply:
good point. will change this to a random one.